test/Microsoft.IdentityModel.TestUtils/KeyingMaterial.cs

```diff
@@ -97,11 +100,11 @@ static public class KeyingMaterial
         public static X509SecurityKey DefaultX509Key_2048_With_KeyId = new X509SecurityKey(DefaultCert_2048) { KeyId = DefaultX509Key_2048_KeyId };
         public static SigningCredentials DefaultX509SigningCreds_2048_RsaSha2_Sha2 = new SigningCredentials(DefaultX509Key_2048, SecurityAlgorithms.RsaSha256Signature, SecurityAlgorithms.Sha256Digest);
         public static X509Certificate2 DefaultAsymmetricCert_2048 = new X509Certificate2(Convert.FromBase64String(DefaultX509Data_2048), CertPassword, X509KeyStorageFlags.MachineKeySet);
-
+#pragma warning restore SYSLIB0057
```

Comment:
This section with `SYSLIB0057` seems to cover too much. Can you move this to only encompass the necessary parts and add more pragmas as needed with some tracking items for them if none already exist?

Reply:
Introducing a helper with the warning disable encompassing only the X509Certificate2 constructor